### DISCUSSION TITLE
install tree-sitter-cli for platform android, architecture arm64

### DISCUSSION THREAD

**Discussion by evascroll** [tree-sitter/tree-sitter#1851] [Q&A] [answered]
hello guys! im new in the computer world! im been trying to install tree-sitter in my tab s8 termux for sometime without any sucess, im setting up my nvim and is the only missing puzzle left, i dont like to ask but its seen that i have hit a wall, any tips or help will be appreciated, thanks

**Comment by inneroot** [comment]
Same here. Testing nvim programmin setup inside pico4 vr, it based on andoid. So same situation here: android->termux terminal.

**Comment by amaanq** [comment]
Android doesn't have support for a rust compiler, you'll have to cross-compile tree-sitter-cli for Android and drop the binary in your path

**Comment by ahlinc** [comment, marked as answer]
Termux on Android has Tree-sitter CLI in its [packages](https://github.com/termux/termux-packages/blob/master/packages/libtreesitter/build.sh) and it can be installed by `pkg install libtreesitter`
But I don't know what architectures are supported on my [aarch64](https://github.com/tree-sitter/tree-sitter/issues/2196#issuecomment-1502365840) it works well.
